src/platform/permissions.ts

```diff
@@ -41,7 +41,7 @@ export const assignUserPermission = (
 
 // api key helpers
 
-export const getOrInsertApiKey = (
+const getOrInsertApiKey = (
```

Comment:
This is not actually used externally anywhere (I had to search for any usage in order to work out the $select) so I removed it to make that clearer